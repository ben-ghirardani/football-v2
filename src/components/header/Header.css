.header {
  width: 100%;
  text-align: left;
  margin: 0px;
  padding: 0px;
  color: white;
  font-size: 40px;
  background-color: rgb(29,39,49);
  font-family: Arial, Helvetica, sans-serif;
}