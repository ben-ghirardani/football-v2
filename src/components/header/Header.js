import React from 'react';
import './Header.css'; 

export default function header () {
  return <div className="header">Football? Yes!</div>
}