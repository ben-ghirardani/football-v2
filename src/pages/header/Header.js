import React, { Component } from 'react';
import './header.css'

// sits inside Table with Body
export default class Header extends Component {

  render() {
    return (
      <tr>
        <th>Pos.</th>
        <th>Team</th>
        <th>MP</th>
        <th>W</th>
        <th>D</th>
        <th>L</th>
        <th>GF</th>
        <th>GA</th>
        <th>GD</th>
        <th>Pts</th>
      </tr>
    )
  }

}