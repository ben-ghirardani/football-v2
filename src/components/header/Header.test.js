import React from 'react';
import Header from './header';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Header/>);
});

describe('Header tests', () => {

  it('Header renders as expected', ()=> {
    expect(wrapper).toMatchSnapshot();
  })

  it('should return a div', () => {
    expect(wrapper.find('div')).toHaveLength(1);
  });

  it('should say Football? Yes!', () => {
    expect(wrapper.find('div').text()).toEqual('Football? Yes!');
  });

});