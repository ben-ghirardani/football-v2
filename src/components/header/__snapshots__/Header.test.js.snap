// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests Header renders as expected 1`] = `ShallowWrapper {}`;

exports[`Header tests matches snapshot 1`] = `ShallowWrapper {}`;

exports[`matches snapshot 1`] = `ShallowWrapper {}`;
